README.md

```diff
@@ -156,6 +160,39 @@ See https://ariejan.net/2010/06/10/cherry-picking-specific-commits-from-another-
     - Click "Create pull request"
     - Add a description of the pull request and start the pull request description with the story or epic ID, `CI-XXX`
 
+## Start Rails server
+
+To start a Rails server with React, you need to  use the command:
+
+`rake assets:precompile`
+
+
+NOTE: `rails s` - is not used
+
+To start the Rails server use: `./bin/dev`
+
+otherwise you won't be able to see your updated CSS and JavaScript 
+
+## OpenSSL::Cipher::CipherError
+
+When you create encrypted credentials in Rails 7, the contents of the `config/credentials.yml.enc` file are encrypted using a master key that is stored in the `config/master.key` file. This encryption process uses the `OpenSSL library`, and if there is a problem with `OpenSSL`, you might see the `OpenSSL::Cipher::CipherError` error.
+
+### Solution
+
+To address the `OpenSSL::Cipher::CipherError` error in Rails application, you can follow these steps:
+
+    - Remove the `config/credentials.yml` file from the project directory.
```

Comment:
For me it was credentials.yml.enc so maybe change this to "Remove `config/credentials.yml` or `config/credentials.yml.enc` if present."